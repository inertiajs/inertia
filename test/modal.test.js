import Modal from '../src/modal'

it('shows modal', () => {
  Modal.show('<p>Hello World</p>')

  expect(Modal.modal).toMatchSnapshot()
})

it('displays html content', () => {
  Modal.show('<p>Hello World</p>')

  const iframe = Modal.modal.firstChild

  expect(iframe.contentDocument.body.innerHTML).toBe('<p>Hello World</p>')
})
