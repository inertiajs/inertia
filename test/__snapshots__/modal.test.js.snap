// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows modal 1`] = `
<div
  style="position: fixed; width: 100vw; height: 100vh; padding: 50px; box-sizing: border-box; background-color: rgba(0, 0, 0, 0.6); z-index: 200000;"
>
  <iframe
    style="background-color: white; border-radius: 5px; width: 100%; height: 100%;"
  />
</div>
`;
